Gemfile.lock

```diff
@@ -399,5 +399,8 @@ DEPENDENCIES
   webmock (~> 3.8.3)
   yard
 
+RUBY VERSION
+   ruby 2.6.6p146
+
```

Comment:
Is this Ruby version bump meant to be included?

Reply:
Bloody rvm. I'll sort this out.